pkg/model/dev.go

```diff
@@ -202,6 +203,12 @@ type PersistentVolumeInfo struct {
 	Size         string `json:"size,omitempty" yaml:"size,omitempty"`
 }
 
+// InitContainer represents the initial container
+type InitContainer struct {
+	Image     string               `json:"image,omitempty" yaml:"image,omitempty"`
```

Comment:
For the usecase where the user is going to push our image into their private repository, how are we going to communicate to the user which bin image we're using?

Reply:
`okteto up` will fail with a clear message